### ISSUE TITLE
@nestjs/throttler v5 ttl unit changed from second to milisecond

### ISSUE THREAD

**Issue by Diluka** [kkoomen/nestjs-throttler-storage-redis#1380]
ref: https://github.com/nestjs/throttler/blob/master/CHANGELOG.md

**Comment by kkoomen** [comment]
Thanks a lot for notifying me. Luckily I had time today so I immediately fixed it. See the corresponding PR.

**Comment by kkoomen** [comment]
v0.4.0 is on NPM https://www.npmjs.com/package/nestjs-throttler-storage-redis

**Comment by stevenolay** [comment]
@kkoomen I appreciate you rushing this one out. However, seems you have NestJS 10 as a hard requirment in the package version. NestJS Throttler v5 is compatible with Nest 7 and up. 

https://www.npmjs.com/package/@nestjs/throttler?activeTab=code

```
  "peerDependencies": {
    "@nestjs/common": "^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0",
    "@nestjs/core": "^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0",
    "reflect-metadata": "^0.1.13"
  },
```

I cannot upgrade to Nest 10 yet because it requires Node 16 and we currently run on Node 14. Can you release another version of this package that supports the same Nest versions as @nestjs/throttler v5?

Thanks!

**Comment by kkoomen** [comment]
@stevenolay Thanks, I honestly completely forgot about this, because I do these things in-between my own busy schedule. I just released v0.4.1 on NPM which contains this fix. Should be good to go now.
